src/domain/settlementWindow/index.js

```diff
@@ -34,16 +34,25 @@ const hasFilters = require('./../../utils/truthyProperty')
 const Producer = require('@mojaloop/central-services-stream').Util.Producer
 const KafkaUtil = require('@mojaloop/central-services-shared').Util.Kafka
 const SettlementWindowModel = require('../../models/settlementWindow')
+const SettlementWindowContentModel = require('../../models/settlementWindowContent')
 const StreamingProtocol = require('@mojaloop/central-services-shared').Util.StreamingProtocol
 const Uuid = require('uuid4')
 
 module.exports = {
   getById: async function (params, enums) {
-    const settlementWindow = await SettlementWindowModel.getById(params, enums)
+    const settlementWindow = await SettlementWindowModel.getById(params) // check if params is correct
```

Comment:
What is the purpose of this comment. Is it a TODO or it's been addressed already?

Reply:
It has been resolved , removed comment